src/content/en/2020/css.md

```diff
@@ -1,87 +1,96 @@
 
-For certain metrics, looking at the CSS AST was not enough. We wanted to look at [SCSS](https://sass-lang.com/) wherever it was provided via sourcemaps as it shows us what developers *need* from CSS that is not yet possible, whereas studying CSS shows us what developers currently use that is. For that, we had to use a *custom metric*, i.e. JS code that runs in the crawler when it visits a given page. We could not use a proper SCSS parser as that could slow down the crawl too much, so we had to resort to [regular expressions](https://github.com/LeaVerou/css-almanac/blob/master/runtime/sass.js) (*oh, the horror!*). Despite the crude approach, we got [a plethora of insights](#sass)!
+For certain metrics, looking at the CSS AST was not enough. We wanted to look at [SCSS](https://sass-lang.com/) wherever it was provided via sourcemaps as it shows us what developers need from CSS that is not yet possible, whereas studying CSS shows us what developers currently use that is. For that, we had to use a *custom metric*—JavaScript code that runs in the crawler when it visits a given page. We could not use a proper SCSS parser as that could slow down the crawl too much, so we had to resort to [regular expressions](https://github.com/LeaVerou/css-almanac/blob/master/runtime/sass.js) (*oh, the horror!*). Despite the crude approach, we got [a plethora of insights](#sass)!
 
-Custom metrics were also used for part of the [Custom properties analysis](#custom-properties). While we can get a lot of information about custom property usage from the stylesheets alone, we cannot build a dependency graph without being able to look at the DOM tree for context, as custom properties are inherited. Looking at the computed style of the DOM nodes also gives us information like what kinds of elements each property is applied to, and which of them are [registered](https://developer.mozilla.org/en-US/docs/Web/API/CSS/RegisterProperty), information that we also cannot get from the stylesheets.
+{# TODO (Authors): it may be obvious to most people who use it (I don't) but is it worth commenting on difference between SCSS and Sass? They are used interchangeably in some places but differentiated in others, so think worth covering here. #}
```

Comment:
@LeaVerou, @svgeesus, @rachelandrew - thoughts on this?

Reply:
This is now inaccurate, since we didn't use custom metrics only for SCSS but also for custom properties. The reference to that custom metric appears to now be removed?

Reply:
Ignore the bit in red. GitHub struggles to show the actual changes when there are large edits like there are here. so concentrate on the bit in green. The custom metrics and custom properties bit is still in there (but in a different place) so fi you want to suggest a change to that then click on the Files tab, find it and click on the line to suggest a change.

In meantime can you comment on the lines in green?

Reply:
Thanks. I added this, but actually done in the Sass section rather than here.